src/router/routes.js

```diff
@@ -28,6 +28,7 @@ export default [
 	{ path: '/lend/filter', component: () => import('@/pages/Lend/Filter/LendFilterPage') },
 	{ path: '/page-two', component: () => import('@/pages/PageTwo') },
 	{ path: '/portfolio/lending-stats', component: () => import('@/pages/LendingStats/LendingStatsPage') },
+	{ path: '/checkout/thanks', component: () => import('@/pages/Thanks/Thanks') },
```

Comment:
It's strongly recommended that components are named with multiple words: https://vuejs.org/v2/style-guide/#Multi-word-component-names-essential (`ThanksPage.vue` would work fine)

Reply:
File name updated in order to prevent conflicts with existing or future HTML elements. Thanks bringing this up Shua.